sql/core/src/test/scala/org/apache/spark/sql/MathExpressionsSuite.scala

```diff
@@ -198,6 +198,15 @@ class MathExpressionsSuite extends QueryTest {
     testOneToOneMathFunction(rint, math.rint)
   }
 
+  test("round") {
+    checkAnswer(
```

Comment:
right now there is only one test for sql expression, no test for the data frame function you added to functions.scala. Just add one more test case that uses dataframes.